waltz-test/src/main/python/waltz_ducktape/services/cli/performance_cli.py

```diff
@@ -26,6 +26,7 @@ def producer_test_cmd(self, txn_size, txn_per_thread, num_thread, interval, lock
             --cli-config-path <the path to cli config file> \
             --lock-pool-size <size of lock pool> \
             --num-active-partitions <number of partitions to interact with>
+            --mount_from_latest client will be initialized with up-to-date water mark from waltz server
```

Comment:
change the description to `Waltz Client would be mounted from the latest HighWaterMark (for a partition) on Waltz`